reference/rebac_admin_backend.md

```diff
@@ -0,0 +1,34 @@
+# ReBAC admin backend
+
+The ReBAC Admin API is a REST API that provides various endpoints to query or
+manipulate relationships in JAAS ReBAC authorisation model.
+
+```{tip}
+For an explanation on Relationship-Based Access Control (ReBAC) check out {doc}`this <../explanation/authorisation>` explanation topic.
+```
+
+```{tip}
+As a reference on JAAS authorisation model, check out {doc}`this <../reference/authorisation_model>` reference topic.
+```
+
+## OpenAPI specification
+
+The OpenAPI spec can be found at `https://<jimm-deployment>/rebac/v1/swagger.json`
```

Comment:
This is the only issue I could spot, this link is a code block instead of a link.

Reply:
It's what is needed for it to render correctly. It's what we had before as well.